app/developers/page.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import Image from 'next/image';
+import { DevBiosTypes } from './interface';
+import { Devs } from './data';
+import SocialLink from '@/components/SocialLink';
+import {
+  LinkedInIcon,
+  GitHubIcon,
+  PortfolioIcon,
+} from '@/public/assets/misc_icons';
+
+export default function Page() {
+  return (
+    <main className='pt-32 grid place-items-center gap-5 max-w-screen-2xl w-full mx-auto'>
+      <h1 className='text-5xl font-bold '>Meet The Team!</h1>
+      <DevBios Devs={Devs} />
+    </main>
+  );
+}
+
+const DevBios = ({ Devs }: DevBiosTypes) => {
+  const items = Devs.map((i) => (
```

Comment:
```suggestion
  const items = Devs.map((dev: DevTypes, index: number) => (
```

then maybe we can do a ` const key = index + 1` so we dont mess up the logic below and we dont have to declare `id` in the type